src/index.js

```diff
@@ -621,6 +621,11 @@ class Analytics {
     }
 
     const items = this.queue.splice(0, this.flushAt);
+    // Do not proceed in case the items array is empty
```

Comment:
if this fix resolves the issue we need it also in service worker for v1.1 and v3